server/model_server/utils.py

```diff
@@ -10,3 +10,25 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         sys.stdout.close()
         sys.stdout = self._original_stdout
+
+# https://medium.com/pythonhive/python-decorator-to-measure-the-execution-time-of-methods-fa04cb6bb36d
+def timeit(method):
```

Comment:
Can we put this util function in `hysia.utils` as a general utility function?

Reply:
Done

Move the file to `hysia.utils.perf`